dom.go

```diff
@@ -225,7 +225,11 @@ func (h *HTML) Restore(old ComponentOrHTML) {
 		panic("vecty: only HTML may have UnsafeHTML attribute")
 	}
 	if h.tag != "" {
-		h.Node = js.Global.Get("document").Call("createElement", h.tag)
+		if h.namespace != "" {
+			h.Node = js.Global.Get("document").Call("createElementNS", h.namespace, h.tag)
+		} else {
+			h.Node = js.Global.Get("document").Call("createElement", h.tag)
+		}
 	} else {
 		h.Node = js.Global.Get("document").Call("createTextNode", h.text)
 	}
```

Comment:
Yeah, not sure about this one - I haven't really got my head around testing for GopherJS to do some benching.  I think I'll just make this change for readability and let @slimsag weigh in if he'd prefer alternative syntax.